client/src/components/Header.jsx

```diff
@@ -1,24 +1,106 @@
-// import React from 'react';
+import React from 'react';
+import Stars from './Stars.jsx';
+import BarGraph from './BarGraph.jsx';
+import style from './style.css';
 
-// // const Header = (props) => (
-// //   <div className="header-container">
-// //     <h1>What 260 People Are Saying</h1>
-// //     <p>Overall ratings and reviews</p>
-// //   </div>
-// // )
+const findOverallRatingOfRestaurant = (list) => {
+  let sum = 0;
+  list.forEach(review => {
+    sum += review.overall_rating;
+  });
+  return parseFloat((sum/list.length).toFixed(1));
+};
 
-// class Header extends React.Component {
-//   constructor(props) {
-//     super(props);
-//     this.state = {
-//       title: ""
-//     }
-//   }
+const findFoodRatingOfRestaurant = (list) => {
+  let sum = 0;
+  list.forEach(review => {
+    sum += review.food_rating;
+  })
+  return parseFloat((sum/list.length).toFixed(1));
+};
 
-//   render(){
+const findServiceRatingOfRestaurant = (list) => {
+  let sum = 0;
+  list.forEach(review => {
+    sum += review.service_rating;
+  })
+  return parseFloat((sum/list.length).toFixed(1));
+};
 
-//   }
+const findAmbienceRatingOfRestaurant = (list) => {
+  let sum = 0;
+  list.forEach(review => {
+    sum += review.ambience_rating;
+  })
+  return parseFloat((sum/list.length).toFixed(1));
+};
 
-// }
+const wouldRecommend = (list, listLength) => {
+  let arr = [];
+  let sum = 0;
+  list.forEach(review => {
+    if (review.overall_rating >= 3) {
+      arr.push(1);
+    }
+  })
+  arr.forEach(count => {
+    sum += count;
+  })
+  return Math.ceil((sum/listLength)*100);
+};
 
-// export default Header;
\ No newline at end of file
+const Header = (props) => {
```

Comment:
You're only using the restaurantInfo prop, you can destructure by doing `const Header = ({ restaurantInfo }) => {` here and then changing `props.restaurantInfo` to just `restaurantInfo`